src/ui/features/onboarding/FAQ/FAQ.tsx

```diff
@@ -8,26 +8,85 @@ import { Spacer } from 'src/ui/ui-kit/Spacer';
 import { TextAnchor } from 'src/ui/ui-kit/TextAnchor';
 import { apostrophe } from 'src/ui/shared/typography';
 import { useWindowSizeStore } from 'src/ui/shared/useWindowSizeStore';
-import lockIconSrc from 'jsx:src/ui/assets/lock.png';
+import lockIconSrc from 'src/ui/assets/lock.png';
 import { Stack } from 'src/ui/ui-kit/Stack';
 import keyIconSrc from '../assets/key.png';
 import dialogIconSrc from '../assets/dialog.png';
 import metamaskIconSrc from '../assets/metamask.png';
+import phantomIconSrc from '../assets/phantom-wallet-icon.svg';
 import metamaskInstructionSrc from '../assets/metamask_instruction.png';
 import walletIconSrc from '../assets/wallet2.png';
 import { SidePanel } from '../shared/SidePanel';
 import * as styles from './styles.module.css';
 
+function PhantomWalletInstructionPanel({
+  show,
+  onDismiss,
+}: {
+  show: boolean;
+  onDismiss: () => void;
+}) {
+  return (
+    <SidePanel show={show} onDismiss={onDismiss}>
+      <VStack gap={20}>
+        <div className={styles.faqIcon}>
+          <img src={phantomIconSrc} style={{ width: 20, height: 20 }} />
+        </div>
+        <VStack gap={16}>
+          <UIText kind="body/accent">
+            Where can I find my recovery phrase or private key?
+          </UIText>
+          <UIText kind="body/regular">
+            1. Open your Phantom wallet extension
+          </UIText>
+
+          <UIText kind="body/regular">
+            2. Click the top-left icon to open the side menu, then tap the
+            pencil icon to manage accounts.
+          </UIText>
+          <div>
+            <img
+              style={{ width: 320 }}
+              src="https://s3.us-east-1.amazonaws.com/cdn.zerion.io/images/dna-assets/phantom-instruction-screenshot_2x.png"
```

Comment:
We can simplify these links to `https://cdn.zerion.io/images***` but this is not critical :)